docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
 
-You can only set the static parameters of a BE by changing them in the corresponding configuration file **be.conf**, and restart the BE to allow the changes to take effect.
+##### dump_log_modules
 
-BE static parameters are as follows.
+- **Default:** query
+- **Description:** The modules for which StarRocks generates dump log entries. By default, StarRocks generates dump logs for the query module. Separate the module names with a comma (,) and a space.
+
+##### dump_log_roll_interval
+
+- **Default:** DAY
+- **Description:** The time interval at which StarRocks rotates dump log entries. Valid values: `DAY` and `HOUR`.
+  - If this parameter is set to `DAY`, a suffix in the `yyyyMMdd` format is added to the names of dump log files.
+  - If this parameter is set to `HOUR`, a suffix in the `yyyyMMddHH` format is added to the names of dump log files.
+
+##### dump_log_roll_num
+
+- **Default:** 10
+- **Description:** The maximum number of dump log files that can be retained within each retention period specified by the `dump_log_roll_interval` parameter.
+
+##### dump_log_delete_age
+
+- **Default:** 7d
+- **Description:** The retention period of dump log files. The default value `7d` specifies that each dump log file can be retained for 7 days. StarRocks checks each dump log file and deletes those that were generated 7 days ago.
+
+#### Server
+
+##### frontend_address
+
+- **Default:** 0.0.0.0
+- **Description:** The IP address of the FE node.
+
+##### priority_networks
+
+- **Default:** Empty string
+- **Description:** Declares a selection strategy for servers that have multiple IP addresses. Note that at most one IP address must match the list specified by this parameter. The value of this parameter is a list that consists of entries, which are separated with semicolons (;) in CIDR notation, such as 10.10.10.0/24. If no IP address matches the entries in this list, an IP address will be randomly selected.
+
+##### http_port
+
+- **Default:** 8030
+- **Description:** The port on which the HTTP server in the FE node listens.
+
+##### http_backlog_num
+
+- **Default:** 1024
+- **Description:** The length of the backlog queue held by the HTTP server in the FE node.
+
+##### cluster_name
+
+- **Default:** StarRocks Cluster
+- **Description:** The name of the StarRocks cluster to which the FE belongs. The cluster name is displayed for `Title` on the web page.
+
+##### rpc_port
+
+- **Default:** 9020
+- **Description:** The port on which the Thrift server in the FE node listens.
+
+##### thrift_backlog_num
+
+- **Default:** 1024
+- **Description:** The length of the backlog queue held by the Thrift server in the FE node.
+
+##### thrift_server_max_worker_threads
+
+- **Default:** 4096
+- **Description:** The maximum number of worker threads that are supported by the Thrift server in the FE node.
+
+##### thrift_client_timeout_ms
+
+- **Default:** 5000
+- **Description:** The length of time after which idle client connections time out. Unit: ms.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/6d014f84-e39c-4ef1-8998-7473e5d38a06)
Move "Unit: ms." up preceding the "Default".